src/main/java/nl/cherement/jak/entity/CardEntity.java

```diff
@@ -20,10 +20,10 @@
     private String description;
 
 	@Column(name="priority")
-    private Long priority;
+    private long priority;
```

Comment:
Ik dacht dat je deze juist naar Long ging veranderen?

Reply:
In de komende commit is dit opgelost